.changeset/unlucky-hats-exist.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin": major
+---
+
+Move the ScopeIndicator from the ToolbarBreadcrumbs to the Toolbar
```

Comment:
Isn't this an internal change?

Reply:
Not really because the slots and class keys change

Reply:
I haven't thought about this as much as you have. So it's okay for me either way.